libflux/flux/Cargo.toml

```diff
@@ -13,12 +13,17 @@ default = ["strict"]
 strict = []
 
 [dependencies]
+comrak = "0.10.1"
 flux-core = { path = "../flux-core" }
 flatbuffers = "0.6.0"
+getrandom = { version = "0.2", features = ["js"] }
```

Comment:
Why is this dependency added? I don't see it used anywhere.

Reply:
The other deps I can see were part of the fluxdoc crate so it makes sense why they were added.

Reply:
Ah I see the WASM compatibility note above, can you add a comment about why the `getrandom` crate is needed? That way its clear to future us.

Reply:
`getrandom` is a dependency of the tera crate, and it was causing the build to fail because it wasn't compatible with WASM. Adding this line allows us to compile to the `wasm32-unknown-unknown` target, as explained here: https://docs.rs/getrandom/0.2.3/getrandom/#webassembly-support

Reply:
I think I forgot to refresh the page before I responded and didn't see your newer comments, haha. Yes, I'll add a comment.